### PR TITLE
Updates to Makefile to work with mapper

### PR DESCRIPTION
**Pull request by norabarlow**

### CODE REVIEW

**Comment by norabarlow** [comment]
@joyliu37 can we merge this PR?

File: cfunc/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@ COREIR_DIR ?= ../../coreir
 
 #COREIR_DIR = /nobackup/joeyliu/aha/coreir
 INCLUDE_DIR += $(COREIR_DIR)/include
-COREIR_LD_FLAGS = -L$(COREIR_DIR)/lib -Wl,-rpath,$(COREIR_DIR)/lib -lcoreir-commonlib -lcoreir -lcoreirsim -lcoreir-float
```
Comment (joyliu37) [inline cfunc/Makefile]: @norabarlow  Why did you remove the other coreir lib?@

Comment (norabarlow) [reply cfunc/Makefile]: I don't quite remember but it was what worked. Do you need them? I can see it adding them back in will make it work.

Comment (joyliu37) [reply cfunc/Makefile]: Umm, it broke my test since I need the coreIR simulator and circuit instance from commonlib. No worries, I will add it since I have already accept the merge.
